src/Parser/Paladin/Retribution/Modules/Features/Checklist/Module.js

```diff
@@ -30,6 +32,8 @@ class Checklist extends Analyzer {
     inquisition: Inquisition,
     righteousVerdict: RighteousVerdict,
     shieldOfVengeance: ShieldOfVengeance,
+    relentlessInquisitor: RelentlessInquisitor,
+    //relentlessInquisitorStackHandler: RelentlessInquisitorStackHandler,
   };
```

Comment:
Commented out here and in the import?

Reply:
Yeah, wasn't sure if it was necessary to import it in the checklist since it was in the combatlogparser. I'll get that removed.